src/providers/aws.rs

```diff
@@ -407,7 +407,7 @@ where
                 regions.insert(l.region.clone(), awsregion);
             }
 
-            let region_span = tracing::debug_span!("region", name = %l.region);
+            let region_span = tracing::info_span!("region", region = %l.region);
```

Comment:
I wonder if we should move this up above the `if`, and then not include `region` in the debug span that gets created inside the if (note that you can `Clone` spans).